tests/step-011.sh

```diff
@@ -2,6 +2,10 @@
 # Test:
 #   Direct template execution: test pre-commit hooks
 
+# Fake a default installation directory (for tests which do not install)
+mkdir -p ~/.githooks
```

Comment:
These tests need to keep working without the config being set!
When we don't have the config, they need to default to `~/.githooks`